hw3/output/Words.group_attr.1D

```diff
@@ -0,0 +1,241 @@
+HISTORY_NOTE = [mli@Ringo-Lite.frontier.com: Fri Aug 18 19:14:42 2017] {AFNI_17.2.06:macosx_10.7_local} 3dttest++ -mask full_mask.group+tlrc -prefix Words.group -setA 'betas/stats*+tlrc.HEAD[Words#0_Coef]'\n[mli@Ringo-Lite.frontier.com: Fri Aug 18 19:29:06 2017] {AFNI_17.2.06:macosx_10.7_local} 3drefit -atrstring AFNI_CLUSTSIM_NN1_1sided file:cluster.NN1_1sided.niml -atrstring AFNI_CLUSTSIM_MASK file:cluster.mask -atrstring AFNI_CLUSTSIM_NN2_1sided file:cluster.NN2_1sided.niml -atrstring AFNI_CLUSTSIM_NN3_1sided file:cluster.NN3_1sided.niml -atrstring AFNI_CLUSTSIM_NN1_2sided file:cluster.NN1_2sided.niml -atrstring AFNI_CLUSTSIM_NN2_2sided file:cluster.NN2_2sided.niml -atrstring AFNI_CLUSTSIM_NN3_2sided file:cluster.NN3_2sided.niml -atrstring AFNI_CLUSTSIM_NN1_bisided file:cluster.NN1_bisided.niml -atrstring AFNI_CLUSTSIM_NN2_bisided file:cluster.NN2_bisided.niml -atrstring AFNI_CLUSTSIM_NN3_bisided file:cluster.NN3_bisided.niml Words.group+tlrc~
```

Comment:
Fun fact - Ringo means apple in Japanese and it also happens to be my favorite Japanese singer's name. Hence the name for my laptop.